### DISCUSSION TITLE
How can I solve this error?

### DISCUSSION THREAD

**Discussion by Guille2307** [primefaces/community#988] [PrimeNG] [answered]
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: netberrytask-front@0.0.0
npm ERR! Found: @angular/common@16.2.12
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"^16.2.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"^17.0.0" from primeng@17.3.3
npm ERR! node_modules/primeng
npm ERR!   primeng@"^17.3.3" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! C:\Users\guill\AppData\Local\npm-cache\_logs\2024-01-22T11_22_10_554Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in: C:\Users\guill\AppData\Local\npm-cache\_logs\2024-01-22T11_22_10_554Z-debug-0.log

**Comment by dalenguyen** [comment, marked as answer]
Are you using Angular 16 & PrimenNG 17? I think the version is a mismatch. You can try to add `--force` and see if it breaks your app or use PrimeNg 16.

**Reply by Guille2307** [reply]
Thanks for your answer, I solved it by uploading the Angular version to 17
